bread/layout/components/notification.py

```diff
@@ -76,21 +70,23 @@ def __init__(
                     _class="bx--inline-notification__action-button",
                 )
             )
-        if not hideclosebutton:
-            children.append(
-                htmlgenerator.BUTTON(
+        children.append(
+            hg.If(
+                hideclosebutton,
+                hg.BUTTON(
                     Icon(
                         "close", size=20, _class="bx--inline-notification__close-icon"
                     ),
                     data_notification_btn=True,
                     _class="bx--inline-notification__close-button",
                     aria_label="close",
-                )
+                ),
             )
+        )
         super().__init__(*children, **attributes)
 
 
-class ToastNotification(htmlgenerator.DIV):
+class ToastNotification(hg.DIV):
```

Comment:
Added auto-fade after n seconds for toast notificaiton (was before only true for toast generated out of django messages).